stable_plugins/classical_ml/data_preparation/pandas_preprocessing/schemas.py

```diff
@@ -0,0 +1,271 @@
+    characters: str
+    column: str
+    new_columns: str
+    substring: str
+    new_str: str
+    case: CaseEnum
+    ignore_index: bool = False
+    ascending: bool = False
+    remove_column: bool = False
+
+    def __str__(self):
+        return str(self.__dict__)
+
+
+class SecondInputParametersSchema(FrontendFormBaseSchema):
+    preprocessing_enum = EnumField(
+        PreprocessingEnum,
+        required=True,
+        allow_none=False,
+        metadata={
+            "label": "Preprocessing Type",
+            "description": """Type of preprocessing the file will undergo.\n
+- do nothing: Does nothing to the file, in case the user accidentally clicked the button `add step`, instead of `done`.\n
+- drop na: Removes rows or columns with a certain amount of missing values.\n
+- fill na: Fills in empty entries with a given value.\n
+- drop duplicates: Removes duplicate rows. May only consider certain columns, to determine if two rows are duplicates.\n
+- sort values: Sorts rows by their values in a certain column.\n
+- strip entries: Strips entries in the specified columns by the given characters. The beginning, the end or both of an entry may be stripped.\n
+- split column: Splits a column into, given a substring to split entries by.\n
+- replace: Replaces all occurrences of a string in all entries with another string.\n
+- string case: Transforms entries into upper, lower or title case.""",
+            "input_type": "select",
+        },
+    )
+    axis = EnumField(
+        AxisEnum,
+        required=True,
+        allow_none=False,
+        metadata={
+            "label": "Axis",
+            "description": "Determine if rows or columns which contain missing values are dropped.",
+            "input_type": "select",
+        },
+    )
+    threshold = ma.fields.Integer(
+        required=True,
+        allow_none=False,
+        metadata={
+            "label": "Threshold",
+            "description": "Requires that many non-NA values. Cannot be combined with how. If left empty, then all values may not be NA.",
+            "input_type": "number",
+        },
+        validate=validate.Range(min=0, min_inclusive=True),
+    )
+    subset = ma.fields.String(
+        required=False,
+        allow_none=True,
+        metadata={
+            "label": "Subset",
+            "description": "To be set by javascript.",
+            "input_type": "text",
+        },
+    )
+    fill_value = ma.fields.String(
+        required=False,
+        allow_none=True,
+        metadata={
+            "label": "Value",
+            "description": "The value to fill every na with.",
+            "input_type": "text",
+        },
+    )
+    keep = EnumField(
+        KeepEnum,
+        required=False,
+        allow_none=True,
+        metadata={
+            "label": "Keep",
+            "description": """Determines which duplicates (if any) to keep.\n
+            - first: Drop duplicates except for the first occurrence.\n
```

Comment:
This might trip up the markdown rendering as `"""` does not strip leading indentation.